runtime/lua/vim/lsp/rpc.lua

```diff
@@ -471,7 +471,7 @@ local function start(cmd, cmd_args, handlers, extra_spawn_params)
   local function handle_body(body)
     local decoded, err = json_decode(body)
     if not decoded then
-      on_error(client_errors.INVALID_SERVER_JSON, err)
+      -- on_error(client_errors.INVALID_SERVER_JSON, err)
```

Comment:
@tjdevries was this remove intentionally? Not mentioned in the commit message nor PR, afaict.

Reply:
Ah, I was having some problems with sumneko locally and I think I commented this out to test  something and never uncommented. We can just add this back I'm pretty sure. Sorry about that.

Reply:
np, will do in https://github.com/neovim/neovim/pull/15803